core/src/main/java/com/linecorp/armeria/client/ClientFactoryOption.java

```diff
@@ -214,6 +215,12 @@
     public static final ClientFactoryOption<MeterRegistry> METER_REGISTRY =
             define("METER_REGISTRY", Metrics.globalRegistry);
 
+    /**
+     * The {@link ProxyConfig} which contains proxy related configuration.
+     */
+    public static final ClientFactoryOption<ProxyConfig> PROXY_CONFIG =
+            define("PROXY_CONFIG", ProxyConfig.disabled());
```

Comment:
How about renaming to `ProxyConfig.direct()` and `DirectProxyConfig`, given that JDK uses `DIRECT`?